src/docs/30_mui_components/buttons.mdx

```diff
@@ -124,6 +124,13 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 		<LensToggle color="info" />
 		<LensToggle color="success" />
 		<LensToggle color="warning" />
-		<LensToggle color="error" />
+	</Box>
+	<Box sx={{ display: 'flex', gap: 2, mt: 2, flexWrap: 'wrap' }}>
```

Comment:
I'm not a fan of the hover for the unselected option matching the active color in many of the cases (primary, info, success, warning). Do you think maybe we can try dimming (but not turning off) the active option when hovering over an unselected option?